jfrog-cli/artifactory/cli.go

```diff
@@ -909,14 +909,18 @@ func getSearchFlags() []cli.Flag {
 
 func getSetPropertiesFlags() []cli.Flag {
 	flags := []cli.Flag{
-		cli.StringFlag{
-			Name:  "props",
-			Usage: "[Optional] List of properties in the form of \"key1=value1;key2=value2,...\". Only artifacts with these properties are affected.` `",
-		},
+		getPropertiesFlag("Only artifacts with these properties are affected."),
 	}
 	return append(flags, getPropertiesFlags()...)
 }
 
+func getPropertiesFlag(description string) cli.Flag {
```

Comment:
It looks like this new function is not used to create the props option for **all** commands. See move, copy, delete and more...

Reply:
Done